### DISCUSSION TITLE
Windows? MacOS?

### DISCUSSION THREAD

**Discussion by pgomezg** [openSUSE/daps#696] [Q&A] [answered]
The documentation, which was released for 3.1, just one year ago, states that no "porting" effort had taken place on any of these operating systems (yet).

Is it still the case? Are there any running initiatives on this subject?

**Comment by fsundermeyer** [comment, marked as answer]
This is still the case and there are no plans to port it. A MAC port would theoretically be possible, but a Windows port would not work, since it relies on a a lot of programs only available on Linux. However, since it is a command line tool, the Windows Subsystem for Linux (WSL) is probably a possible solution (have not tested it).
We are also offering a container: https://github.com/openSUSE/daps2docker

HTH
